frontend/src/Battle/index.jsx

```diff
@@ -0,0 +1,198 @@
+import React, { useState } from 'react';
+import {
+  View, StyleSheet, ScrollView, Text, Alert, Button,
+} from 'react-native';
+// import { SERVER_URL } from 'react-native-dotenv';
+import PlanetItem from './PlanetItem';
+import PlayerItem from './PlayerItem';
+import { CardView } from '../common/components';
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    justifyContent: 'center',
+    paddingTop: -10,
+  },
+  planetsContainer: {
+    marginVertical: 20,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    alignItems: 'center',
+  },
+  imageStyle: {
+    height: 64,
+  },
+  scrollView: {
+    flex: 1,
+  },
+});
+
+const PLANETS = ['blue', 'green', 'yellow', 'red', 'purple'];
+
+const mockedUsers = [
```

Comment:
battle map backend is already merged, can you please use real data on this branch?
if not, what is it depending on?

Reply:
Of course, I can connect on this branch.